### ISSUE TITLE
Confusion in binary detection(Track 1).

### ISSUE THREAD

**Issue by nikhilbyte** [hugojair/mocca-at-icmi21#4]
it is said that binary_detection.sol contains the ground truth labels for the binary detection task (track 1) in the training set. There is one line per each of the videos in the Dataset_ComicMischief_Training_Scene_Binary_Annotations.csv file. 

But the Dataset_ComicMischief_Training_Scene_Binary_Annotations.csv has 998 rows and the binary_detection.sol has around 5060 lines. 
Can you please clear this doubt ?

**Comment by hugojair** [comment]
Apologies for this, it was an error on our side. We have updated the *.sol file accordingly, thank you very much for pointing this out
